docs/windows.md

```diff
@@ -59,7 +59,43 @@ and daemons are pre-installed on the Windows Nodes in the demo.
     See details in [Join Windows worker Nodes](#Join-Windows-worker-nodes)
     section.
 
-### Installation
+### Installation as a Service (containerd based runtimes)
```

Comment:
I think Quan had a PR to harmonize the spelling to "Containerd"

Reply:
Indeed. Updated.